src/main/java/org/benetech/servicenet/repository/ActivityRepository.java

```diff
@@ -211,9 +209,10 @@ private Predicate addDateFilter(FiltersActivityDTO filtersActivityDTO, Predicate
         }
 
         predicate = addDateFilter(filtersActivityDTO, predicate, root);
+        Join<ActivityInfo, OrganizationMatch> matchJoin = root.join(ORGANIZATION_MATCHES, JoinType.LEFT);
+        predicate = cb.and(predicate, cb.equal(matchJoin.get(HIDDEN), filtersActivityDTO.getHiddenFilter()));
```

Comment:
is this filter always not null?

Reply:
I did this with default hidden as false. Do you think it would be better not to have default?

Reply:
if there is no possibility to set it to null on the frontend it should be fine